portfolio/src/main/webapp/aboutme.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio | About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class ="content">
+      <!-- navigation menu -->
+      <div class="header">
+        <a href ="aboutme.html">about me</a>
```

Comment:
nit: let's keep spacing for href consistent like`href="your string"`. This applies to all your HTML files.